src/main.c

```diff
@@ -0,0 +1,389 @@
+#define _CRT_SECURE_NO_WARNINGS 1
+
+
+#include "porta.h"
+#include "arith.h"
+#include "common.h"
+#include "log.h"
+#include "inout.h"
+#include "mp.h"
+#include "four_mot.h"
+#include "portsort.h"
+
+
+#define FILENAME_SIZE 1000
+
+FILE *logfile;
+
+
+int main( int argc, char *argv[] )
+{
+    int i, ieq_file, start;
+    char outfname[FILENAME_SIZE];
+    char fname[FILENAME_SIZE];
+    int   poi_file;
+    int   rowl_inar, ierl;
+    int  *indx = (int *)0;
+    int equa_in,ineq_in, ineq_out;
+    FILE *outfp;
+
+
+
```

Comment:
let's remove the extraneous whitespace

Reply:
done (do you want to reserve the right to resolve these convos? i'll hold off on pressing the big green button)